docs/create-packages/Sign-a-Package.md

```diff
@@ -0,0 +1,101 @@
+
+|Package Type|Notes    |
+|------------|---------|
+|Unsigned|The package does not include any signature.|
+|Author|The primary signature is an author signature.|
+|Repo|The primary signature is a repository signature.|
+|Author and Repo|The primary signature is an author signature, and the signature is repository countersigned.|
+
+> [!Note] 
+> Package repositories can host packages of any of these four types, however once NuGet.org implements repository signatures, all packages in NuGet.org will fall in the last two types.
+
+The technical specification for NuGet package signatures can be found in https://github.com/NuGet/Home/wiki/Package-Signatures-Technical-Details. 
+
+### Benefits of author-signed packages
+
+Packages that have been signed with an author certificate provide the strongest end to end validation, since the author signatures guarnatees that the package has not been modified since it leaves the author machine, until it is downloaded into the consumer machine, no matter fro which repository or transport method has been used.
+
+Consumers who demand a locked-down environment can require packages signed with an specific author certificate.
+
+Additionally, it provides an extra authentication mechanism to the publishing mechanism to NuGet.org, since the certificate used to sign must be registered before, as described in the [Registering Certificates](https://github.com/NuGet/Home/wiki/Register-package-signing-certificates) specification.
+
+### Benefits of repository-signed packages
+
+Package repositories who implement repository signatures will provide a signed version of all packages, no matter if the package is author-signed or not. This capability allows to verify the complete dependency graph even for cases where a signed package has a dependency on an unsigned package. 
+
+Packages with the repository signature offer content integrity and proof of origin, even when the package is found in a different mirror/repository.
+
+# How to Author-Sign a package
+
+NuGet package authors who want to sign a package must obtain a X.509 code signing certificate. It’s strongly recommended to obtain the certificate from a public certification authority. The author can also use a certificate from a private certification authority but these signatures will generate a warning if the certificate is not trusted in the machine where the validation will occur.
+
+## Certificate Requirements
+
+Authors require a code signing certificate, these are a special type of certificate that are valid for the `id-kp-codeSigning` purpose [[RFC 5280 section 4.2.1.12](https://tools.ietf.org/html/rfc5280#section-4.2.1.12)], and have a RSA public key length of 2048 bits or higher.
+
+These certificates can be obtaied from a public CA like Symmantec.com, DigiCert.com, GlobalSign.com or Certum.pl 
+> [!Tip]
+> Certum has special offers for [open source projects](https://www.certum.eu/certum/cert,offer_en_open_source_cs.xml).
+
+## Signing with NuGet.exe
```

Comment:
"NuGet.exe" >> "nuget.exe" -- also in the next paragraph. We always use "nuget.exe" in the docs.